src/Nethermind/Nethermind.Network/P2P/Subprotocols/Etha/EthaProtocolFactory.cs

```diff
@@ -0,0 +1,42 @@
+using Nethermind.Blockchain;
+using Nethermind.Logging;
+using Nethermind.Stats;
+
+namespace Nethermind.Network.P2P.Subprotocols.Etha
+{
+    public class EthaProtocolFactory : ProtocolFactoryBase
+    {
+        private readonly IBlockTree _blockTree;
+        private readonly IMessageSerializationService _serializationService;
+        private readonly INodeStatsManager _nodeStatsManager;
+        private readonly ILogManager _logManager;
+
+        public EthaProtocolFactory(
```

Comment:
It's not needed, we can initialize `EthaProtocolHandler` directly where needed. This class can be removed